SwiftStylable/Classes/Style/Dimensions/DimensionCollection.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DimensionCollection.swift
+//  Pods-SwiftStylableExample
+//
+//  Created by Rens Wijnmalen on 04/02/2019.
+//
+
+import UIKit
+
+public class DimensionCollection {
+    
+    private var _dimensionHolders = [String:DimensionHolder]()
+    
+    
+    // -----------------------------------------------------------------------------------------------------------------------
+    //
+    // MARK: - Initializers
+    //
+    // -----------------------------------------------------------------------------------------------------------------------
+    
+    
+    // -----------------------------------------------------------------------------------------------------------------------
+    //
+    // MARK: - Internal methods
+    //
+    // -----------------------------------------------------------------------------------------------------------------------
+    
+    internal func applyData(_ data:[String:Any]) {
+        var dimensionsEntries = data
+        var numParsedDimensions = 1
+        while dimensionsEntries.count > 0 && numParsedDimensions > 0 {
+            numParsedDimensions = 0
+            for (name, dimensionValue) in dimensionsEntries {
```

Comment:
Is het hier niet mooier om de dimensionEntries door middel van een map te verwerken?

Reply:
Ik heb deze manier nu overgenomen van hoe marcel het al deed bij de kleuren zodat dit ook op dezelfde manier wordt gedaan (Ongeveer dezelfde manier van verwerken). Ik zal hier wel een ticket voor aanmaken om te kijken of we het bij beide manieren wellicht iets kunnen optimaliseren.

Reply:
is een beetje outdated voor mijn gevoel, zeker omdat je een `while` gebruikt.